charts/splunk-connect-for-snmp/values.yaml

```diff
@@ -455,8 +537,64 @@ mongodb:
     storageClass: "microk8s-hostpath"
   volumePermissions:
     enabled: true
+  livenessProbe:
+    # The liveness probe checks if the MongoDB instance is running and responsive.
+    exec:
+      # The exec command for the liveness probe to run in the container.
+      command:
+        - mongo
+        - --eval
+        - 'db.adminCommand("ping")'
+    # Number of seconds after the container has started before liveness probes are initiated.
+    initialDelaySeconds: 30
+    # How often (in seconds) to perform the probe.
+    periodSeconds: 10
+
+  readinessProbe:
+    # The readiness probe checks if the MongoDB instance is ready to accept traffic.
+    # This configuration is similar to the liveness probe but with shorter intervals.
+    # It checks if the MongoDB instance is ready to accept traffic. The probe runs every
+    # 5 seconds after an initial delay of 10 seconds.
+    exec:
+    # The exec command for the readiness probe to run in the container.
+    command:
+      - mongo
+      - --eval
+      - 'db.adminCommand("ping")'
+    # Number of seconds after the container has started before readiness probes are initiated.
+    initialDelaySeconds: 10
+    # How often (in seconds) to perform the probe.
+    periodSeconds: 5
 redis:
   architecture: standalone
   auth:
     enabled: false
+  livenessProbe:
```

Comment:
same with redis: https://github.com/bitnami/charts/tree/main/bitnami/redis

Reply:
same changes as in mongo case